addins/CFSample/README.md

```diff
@@ -0,0 +1,12 @@
+# License
+See https://officedev.github.io/custom-functions/LICENSE
+
+# Purpose
+This add-in is for internal testing purposes.It is used to test CustomFunction which contains different sample of Functions like: DialogApi, OfficeStorage, streaming functions etc, along with a Taskpane which contain OfficeStorage API to get/set Token from the Storage. This Addin basically runs CustomFunctions inside the JSRuntime.
```

Comment:
Spell the feature as 'Custom Functions'. Then, use a regular noun 'function' when you reference JavaScript functions.